src/libopensc/reader-pcsc.c

```diff
@@ -101,10 +101,20 @@ void APDU_LOG(u8 *rbuf, uint16_t rsize)
 #define APDU_LOG(rbuf, rsize)
 #endif
 
+#define MAX_PCSC_EVENT_LISTENERS
+#define ARRAY_SIZE(x) (sizeof(x)/sizeof(x[0]))
```

Comment:
we have this helper already in `pkcs11-tool.c`. Would it make sense to move it to some common header file?